source/monitoring.txt

```diff
@@ -212,22 +367,34 @@ Source Connector JMX Metrics
 
    * - **records**
```

Comment:
Question: My understanding is that this page is dedicated to Monitoring and a separate page will be dedicated to Logging. Are all of the metrics in this table related specifically to monitoring and not logging?

Reply:
Yes, these are all attributes in JMX. This page does not discuss logging.